scala-generator/src/main/scala/models/play/files/Routes.scala

```diff
@@ -2,12 +2,14 @@ package scala.models.play.files
 
 import io.apibuilder.generator.v0.models.InvocationForm
 
+import scala.models.Config
+
 object Routes {
 
   def contents(form: InvocationForm): String = {
     val header = scala.models.ApidocComments(form.service.version, form.userAgent).forPlayRoutes + "\n"
-    scala.models.Play2RouteGenerator
-      .invoke(form)
+    new scala.models.Play2RouteGenerator(form, Config.PlayGen2DefaultConfig)
```

Comment:
`Play2RouteGenerator` seems to be `case class`. is `new` necessary?

Reply:
👍 thx good point